src/get_data.py

```diff
@@ -447,6 +457,31 @@ def get_roster(year : int, season : str, game_number : int) -> pd.DataFrame:
     players = pd.DataFrame(players)
     return players
 
+
+def combine_api_scrape_data(api_df : pd.DataFrame, scrape_df : pd.DataFrame, year : int) -> pd.DataFrame:
+    """
+    Parameters 
+    ----------
+    api_df : DataFrame
+        A Pandas DataFrame constructed using the
+        _parse_api_plays() method
+
+    scrape_df : DataFrame
```

Comment:
It's typically convention to define functions/methods with `_function()` only if they are private and not public to the client. Is this the intended usage? Just double checking - if so, we would _typically_ not describe underlying implementation details on public functions, but only what they do. For a private function, it's ok to document internal API/developer level details.

Reply:
I imagine most of these functions will be private, and the public function would be a pipeline type process that calls all these functions in sequence & passes the proper variables between them. We will likely have to have a discussion on how we want to structure all this and if we want to make this a class that has class variables that are shared among methods as opposed to passing variables between the functions. I could be misunderstanding what you're talking about and I may not be making sense either, next time we get together we can hash it all out.